src/main/modules/session/index.ts

```diff
@@ -9,8 +9,12 @@ const { Logger } = require('@hmcts/nodejs-logging');
 export class Session {
   logger = Logger.getLogger('session');
   enableFor(app: Express): void {
-    const redis = new Redis(config.get<string>('secrets.pcs.redis-connection-string'));
+    const redisConnectionString =
+      process.env.REDIS_CONNECTION_STRING || config.get<string>('secrets.pcs.redis-connection-string');
+    this.logger.info(`Connecting to Redis at: ${redisConnectionString}`);
```

Comment:
Do not log the connection string please

Reply:
was only for testing as part of my initial implementation, which i clearly didnt remove :-)